tools/rna_tools/trna_mod/bbduk_wrapper.xml

```diff
@@ -0,0 +1,46 @@
+<tool id="tRNAmodbbduk" name="bbduk_for_tRNAmod" version="0.1.0">
+  <description>Preprocesses reads with bbduk according to best practice for tRNA modification analysis</description>
+  <requirements>
+	<requirement type="package">bbmap</requirement>
+  </requirements>
+  <command>
+	<![CDATA[
+			 #if $input.is_of_type("fastq.gz", "fastqsanger.gz"):
+			 ln -sf "${input}" "${input}.fastq.gz";
+			 ln -sf "${adapter}" "${adapter}.fa";
+			 ln -sf "${output}" "${output}.fastq";
+			 bbduk.sh in=${input}.fastq.gz out=${output}.fastq ref=${adapter}.fa mink=8 ktrim=r k=10 rcomp=t hdist=1 qtrim=rl trimq=25 minlength=50 maxlength=100 overwrite=t	 
+			 #else
+			 ln -sf "${input}" "${input}.fastq";
+			 ln -sf "${adapter}" "${adapter}.fa";
+			 ln -sf "${output}" "${output}.fastq";
+			 bbduk.sh in=${input}.fastq out=${output}.fastq ref=${adapter}.fa mink=8 ktrim=r k=10 rcomp=t hdist=1 qtrim=rl trimq=25 minlength=50 maxlength=100 overwrite=t
+			 #end if
+	]]>
+  </command>
+  <stdio>
```

Comment:
You can replace this by detect_errors: https://docs.galaxyproject.org/en/latest/dev/schema.html#detect-errors